merkle-tree/concurrent/tests/tests.rs

```diff
@@ -698,13 +750,48 @@ async fn test_spl_compat() {
             .to_bytes_be()
             .try_into()
             .unwrap();
+
         concurrent_mt.append(&leaf).unwrap();
         spl_concurrent_mt.append(leaf).unwrap();
         reference_tree.update(&leaf, i).unwrap();
 
         let root = concurrent_mt.root().unwrap();
-        assert_eq!(root, spl_concurrent_mt.get_change_log().root,);
+        assert_eq!(root, spl_concurrent_mt.get_root());
```

Comment:
This is not necessary anymore because of the compare_trees function right?

Reply:
It's not, let me remove before merging